lib/members.js

```diff
@@ -23,6 +23,7 @@ var EventEmitter = require('events').EventEmitter;
 var util = require('util');
 
 var LOG_10 = Math.log(10);
+var MEMBER_STATUSES = ['alive', 'faulty', 'leave', 'suspect'];
```

Comment:
nitpick: make this an object whose value is always true.

i.e. `{ alive: true, faulty: true, ... }`

Then you can do `MEMBER_STATUSES[status]` instead of `indexOf()`